### ISSUE TITLE
onload isn't called on the top level element of the component

### ISSUE THREAD

**Issue by paulsonnentag** [yoshuawuyts/cache-element#10]
It seems like the onload handler isn't called on the top level element. If I move the handler to the .inner element it works just fine.

Here is how my component looks like:

```javascript
const widget = require('cache-element/widget')
const html = require('choo/html')

const myComponent = widget((update) => {
  update(onupdate)

  return html`   
    <div class='outer' onload=${onload} onunload=${onunload}>
      <div class='inner'></div>
    </div> 
  `

  function onupdate (stat, prev, send) {
   // update
  }

  function onload (el) {
    console.log('loaded')
  }

  function onunload () {
    // unload
  }
})

```

**Comment by yoshuawuyts** [comment]
yeah this is a bug indeed - was afraid this might be the case - I should
fix this

**Comment by yoshuawuyts** [comment]
Hm yeah so to update: we can't get the `onload` property from `bel` elements because they're not passed in - should probably make multiple onload handlers stack / expose it as a property on the element or something

**Comment by yoshuawuyts** [comment]
e.g. https://github.com/shama/bel/blob/master/index.js

**Comment by yoshuawuyts** [comment]
This should be solved in `v3.0.0` I think
